sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/join/UnionCoderTest.java

```diff
@@ -39,9 +37,4 @@ public void testCoderIsSerializable() {
     CoderProperties.coderSerializable(UnionCoder.of(ImmutableList.<Coder<?>>of(
         StringUtf8Coder.of(), DoubleCoder.of())));
   }
-
-  @Test
-  public void testCoderIsSerializableWithWellKnownCoderType() {
```

Comment:
The test is some kind of copy/paste error, not actually testing the class in question at all, that would make the SDK depend on runners/core.